assets/src/scss/vendors/_bootstrap.scss

```diff
@@ -3,12 +3,14 @@ $grid-breakpoints: (
   sm: 576px,
   md: 768px,
   lg: 992px,
-  xl: 1200px
+  xl: 1200px,
+  xxl: 1920px,
```

Comment:
The ticket mentions "New breakpoint on extra extra large screen ≥1920px". If we want to match full HD screens, we should set this value a bit lower to account for small amounts of spaces the OS can subtract from the browser window (e.g. sidebar).

Bootstrap itself sets this value to 1400px by default, probably intending to match 1440px and up.

Reply:
I remember we discussed this during the refinement meeting, but somehow we didn't capture it on the ticket :thinking: